apis/v1alpha2/shared_types.go

```diff
@@ -65,7 +65,9 @@ type ParentReference struct {
 	// SectionName is the name of a section within the target resource. In the
 	// following resources, SectionName is interpreted as the following:
 	//
-	// * Gateway: Listener Name
+	// * Gateway: Listener Name. When both Port and SectionName are specified,
```

Comment:
Noticed that this shows up in stable API definitions. Until we have a better way to differentiate, something like this might work:
```suggestion
	// * Gateway: Listener Name. When both Port (experimental) and SectionName are specified,
```

Reply:
Updated.

Reply:
Uh, there's no mention of the experimental status of Port here? Did I miss something?

Reply:
Oops, for some reason my git push did not go through. It's updated now.